src/core/Database.cpp

```diff
@@ -308,6 +308,22 @@ void Database::recycleGroup(Group* group)
      }
 }
 
+void Database::emptyRecycleBin()
+{
+    if (m_metadata->recycleBinEnabled() && m_metadata->recycleBin()) {
+        // destroying direct entries of the recycle bin
+        QList<Entry*> subEntries = m_metadata->recycleBin()->entries();
+        for (Entry* entry : subEntries) {
+            delete entry;
```

Comment:
Is this the appropriate way to get rid of an entry? I would imagine you would have to go through the db object to remove entries and groups officially.